.stdci.yml

```diff
@@ -0,0 +1,6 @@
+---
+Architectures:
+  - x86_64:
+      Distributions: ["el7"]
```

Comment:
The commit message mentions Fedora 28, too. So please either add it here or change the commit message.

Reply:
Done